### PR TITLE
MINOR: change file to column index in page_filter trace log

### PR DESCRIPTION
**Pull request by mapleFU**
## Which issue does this PR close?

No issue, just a minor

## Rationale for this change

Origin log is:

```
"skip page pruning due to lack of indexes. Have offset: {} file: {}",
```

I change it to:

```
"skip page pruning due to lack of indexes. Have offset: {}, column index: {}",
```

Other is fmt change.

## What changes are included in this PR?

Change the log

## Are these changes tested?

no

## Are there any user-facing changes?

log wording change

### CODE REVIEW

File: datafusion/core/src/datasource/physical_plan/parquet/page_filter.rs [hunk 1]
```diff
@@ -147,17 +147,19 @@ impl PagePruningPredicate {
...
-                    trace!(
-                    "skip page pruning due to lack of indexes. Have offset: {} file: {}",
+        let (file_offset_indexes, file_page_indexes) = match (
+            file_offset_indexes,
+            file_page_indexes,
+        ) {
+            (Some(o), Some(i)) => (o, i),
+            _ => {
+                trace!(
+                    "skip page pruning due to lack of indexes. Have offset: {} column index: {}",
```
Comment (Weijun-H) [inline datafusion/core/src/datasource/physical_plan/parquet/page_filter.rs:157]:
```suggestion
                    "skip page pruning due to lack of indexes. Have offset: {}, column index: {}",
```

**Review by alamb — APPROVED** [review]
Thank you @mapleFU

File: datafusion/core/src/datasource/physical_plan/parquet/page_filter.rs (hunk 1 above)
Comment (mapleFU) [reply datafusion/core/src/datasource/physical_plan/parquet/page_filter.rs]: done

**Comment by mapleFU** [comment]
This patch CI failed in https://github.com/apache/arrow-datafusion/actions/runs/6393777261/job/17353702906?pr=7730 . Is this my fault?

**Comment by mapleFU** [comment]
(oh seems I need to rebase the project)

**Comment by mapleFU** [comment]
Oh i think that's : https://github.com/apache/arrow-datafusion/pull/7733

**Comment by alamb** [comment]
I have merged https://github.com/apache/arrow-datafusion/pull/7733 so iif you rebase again the CI should pass. Thanks @mapleFU

**Comment by mapleFU** [comment]
I've rebase and update the description now.

**Pull request merged**